shared/operation.go

```diff
@@ -106,9 +106,9 @@ func (o *Operation) GetError() error {
 	return nil
 }
 
-func (r *Operation) MetadataAsMap() (*Jmap, error) {
+func (o *Operation) MetadataAsMap() (*Jmap, error) {
 	ret := Jmap{}
-	if err := json.Unmarshal(r.Metadata, &ret); err != nil {
+	if err := json.Unmarshal(o.Metadata, &ret); err != nil {
```

Comment:
Other methods on the Operation struct is using ```func (o *Operation) X Y``` format whereas this guy was using ```func (r *Operation) X Y``` and lint was whining about it, suggesting consistently naming the method receivers.